spec/operators/inspectTime-spec.js

```diff
@@ -0,0 +1,136 @@
+/* globals describe, it, expect, expectObservable, expectSubscription, hot, cold, rxTestScheduler, expectSubscriptions */
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
+var Scheduler = Rx.Scheduler;
+
+describe('Observable.prototype.inspectTime()', function () {
+  it.asDiagram('inspectTime(50)')('should emit the last value in each time window', function () {
+    var e1 =   hot('-a-x-y----b---x-cx---|');
+    var subs =     '^                    !';
+    var expected = '------y--------x-----|';
+
+    var result = e1.inspectTime(50, rxTestScheduler);
```

Comment:
@staltz cool. I copied the tests from [`throttleTime`](https://github.com/ReactiveX/RxJS/blob/master/spec/operators/throttleTime-spec.js) so as to be consistent. perhaps we can clean up the tests in a followup PR.